src/test/FuzzerImpl.cpp

```diff
@@ -503,17 +517,82 @@ isBadOverlayFuzzerInput(StellarMessage const& m)
 
 static void
 attemptToApplyOps(LedgerTxn& ltx, PublicKey const& sourceAccount,
-                  xdr::xvector<Operation> const& ops, Application& app)
+                  xdr::xvector<Operation> const& ops, Application& app,
+                  bool const throwIfTxFails = true)
 {
     auto txFramePtr =
         createFuzzTransactionFrame(sourceAccount, ops, app.getNetworkID());
     txFramePtr->attemptApplication(app, ltx);
 
-    if (txFramePtr->getResultCode() != txSUCCESS)
+    if (throwIfTxFails)
     {
-        throw std::runtime_error(fmt::format(
-            FMT_STRING("Error {} while applying operations while fuzzing"),
-            txFramePtr->getResultCode()));
+        auto const opFrames = txFramePtr->getOperations();
+        if (txFramePtr->getResultCode() != txSUCCESS)
+        {
+            auto const msg = fmt::format(
```

Comment:
Another example of working too hard. Just log the entire transaction result using `xdr_to_string`. This isn't user-facing, it's developer facing.

Reply:
Done.